onnxruntime/featurizers_ops/cpu/count_vectorizer_transfromer.cc

```diff
@@ -40,6 +40,7 @@ void CountVectorizerTransformerImpl(OpKernelContext* ctx) {
     }
   };
   transformer.execute(*input_data, callback);
+  transformer.flush(callback);
```

Comment:
>transformer.flush(callback); [](start = 2, length = 28)

So flush() would invoke callback and overwrite data that was written before? Is this as intended?